pupil-api/yarn.lock

```diff
@@ -17,7 +17,7 @@
   dependencies:
     tslib "^2.2.0"
 
-"@azure/core-amqp@^3.3.0":
+"@azure/core-amqp@^3.1.0":
```

Comment:
Why the downgrade?

Reply:
👀  well that's odd, given i didn't touch that app.

I need to align all of the projects onto TS 4.5.5, so will focus on that next.